README.md

```diff
@@ -296,6 +296,16 @@ env_wrapper:
 
 Note that you can easily specify parameters too.
 
+By default, the environment is wrapped with a `Monitor` wrapper to record episode statistics.
+You can specify arguments to it using `monitor_kwargs` parameter to log additional data that *must* be present
+in the info dictionary.
+For instance, for recording success with goal envs (e.g. `FetchReach-v1`):
+
+```yaml
+monitor_kwargs: dict(info_keywords=('is_success',))
+```
+
+
```

Comment:
I think it could be written more clearly that only the info from the last interaction of the episode is monitored.

Reply:
Something like:


You can specify arguments to it using `monitor_kwargs` parameter to record additional data that *must* be present in the information dictionary of the last interaction of the episode.
For instance, for recording final x position with `Ant-v3`:

 ```yaml
 monitor_kwargs: dict(info_keywords=('x_position',))
 ```

Reply:
(For FetchReach, `info_keywords=('is_success',)` is already the default value)